community/kernel/src/main/java/org/neo4j/kernel/impl/util/DebugUtil.java

```diff
@@ -364,4 +368,78 @@ private static boolean hasTestAnnotation( Method method )
         }
         return false;
     }
+
+    /**
+     * Super simple utility for determining where most time is spent when you don't know where to even start.
+     * It could be used to home in on right place in a test or in a sequence of operations or similar.
+     */
+    public abstract static class Timer
```

Comment:
How is Stopwatch the same thing? This simple utility prints times at different points where it's used. It's the printing which is useful and it's only a utility quite useful when seeing what takes time in a test.